src/openapi.ts

```diff
@@ -147,6 +147,31 @@ export function OpenAPIRouter(options?: RouterOptions): OpenAPIRouterSchema {
         })
       })
     }
+
+    if (options?.aiPlugin && options?.openapi_url !== null) {
+      router.get('/.well-known/ai-plugin.json', (request: IRequest) => {
+        return new Response(
+          JSON.stringify({
+            schema_version: SchemaVersion.V1,
+            api: {
+              type: APIType.OPENAPI,
+              has_user_authentication: false,
+              url: `https://${request.headers.get('host')}/openapi.json`,
```

Comment:
the `openapi.json` here should be replaced with `options?.openapi_url || '/openapi.json'` to account for a different path

Reply:
fixed!